src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task {
+    protected String by;
```

Comment:
It might be good to make variable names more meaningful, to help potential reviewers understand the code better. It will also help with creating more meaningful function names.